PowerEditor/src/WinControls/PluginsAdmin/pluginsAdmin.rc

```diff
@@ -35,8 +35,8 @@ BEGIN
 	PUSHBUTTON      "Update", IDC_PLUGINADM_UPDATE,432,35,57,14
 	PUSHBUTTON      "Remove", IDC_PLUGINADM_REMOVE,432,35,57,14
 	EDITTEXT        IDC_PLUGINADM_EDIT,30,160,220,80,ES_MULTILINE | ES_AUTOVSCROLL | ES_READONLY | WS_BORDER | WS_VSCROLL
-	RTEXT           "Plugin list version: ", IDC_PLUGINLIST_VERSION_STATIC,350,230,120,14
-	LTEXT           "", IDC_PLUGINLIST_VERSIONNUMBER_STATIC,470,230,30,14
-	RTEXT           "Plugin list repository",IDC_PLUGINLIST_ADDR,368,245,120,14
+	RTEXT           "Plugin list version: ", IDC_PLUGINLIST_VERSION_STATIC,350,242,120,14
+	LTEXT           "", IDC_PLUGINLIST_VERSIONNUMBER_STATIC,470,242,30,14
+	RTEXT           "Plugin list repository",IDC_PLUGINLIST_ADDR,368,257,120,14
```

Comment:
Why not?
But I would do the following:
![image](https://user-images.githubusercontent.com/90293/192109049-a617d16c-0cb0-4ec7-be13-8adcbeea9693.png)

Reply:
@donho Done!